shelter/README.md

```diff
@@ -28,6 +28,17 @@ git clone https://github.com/alibaba/inclavare-containers
 
 2. Prepare the dependence libs required by shelter
 
+For EPID RA
+
+```shell
+cd $WORKSPACE/inclavare-containers/shelter
+make
+```
+
+For DCAP RA
+
+Please refer to this guide to install DCAP. Note: If your platform is pre-product SGX platform (SBX), please follow this guide to resolve the quote verification problem on SBX platforms: https://github.com/alibaba/inclavare-containers/blob/master/hack/use-sbx-platform/README.md.
```

Comment:
Please add the url for `this guide`

Reply:
You should write [this guide](URL) Markdown style.

Reply:
ok, thx, some missing.